frontend/lib/search/search_page.dart

```diff
@@ -20,42 +20,47 @@ class _SearchPageState extends State<SearchPage> {
 
   @override
   Widget build(BuildContext context) {
-    var currentCoordinatesInput = _coordinates;
-    return Stack(children: [
-      CustomScrollView(
-        slivers: [
-          SliverSearchNavigationBar(
+    final currentCoordinatesInput = _coordinates;
+    return Stack(
+      children: [
+        CustomScrollView(
+          slivers: [
+            SliverSearchNavigationBar(
               onChanged: (text) => setState(() {
-                    searchFieldText = text;
-                  })),
-          FilterBar(onCategoryPress: _onCategoryPress),
-          SliverToBoxAdapter(
+                searchFieldText = text;
+              }),
+            ),
+            FilterBar(onCategoryPress: _onCategoryPress),
+            SliverToBoxAdapter(
               child: Padding(
-                  padding: const EdgeInsets.all(8),
-                  child: Row(children: [
+                padding: const EdgeInsets.all(8),
+                child: Row(
+                  children: [
                     Text(
                       "Suchresultate".toUpperCase(),
                       style: const TextStyle(color: Colors.grey),
                     ),
-                    const Expanded(
-                        child: Padding(
-                            padding: EdgeInsets.only(left: 8),
-                            child: Divider()))
-                  ]))),
-          ResultsLoader(
+                    const Expanded(child: Padding(padding: EdgeInsets.only(left: 8), child: Divider()))
```

Comment:
Not sure if I understand the question. We did not have a linting/format rule to force each child to have its own line. This was rather just the case because of the shorter previous line length. AFAIK there are no formatting options except from line length, so probably not possible to force children to have its one line.

Reply:
I see yes, seems like line length is the only which causes children to go onto a new line most of the cases. I did not find such a linting rule.